src/components/BetaInfoBox.jsx

```diff
@@ -0,0 +1,44 @@
+/* global _ */
+import React, { useState } from 'react';
+import Alert from './ui/Alert';
+import { Button } from './ui';
+import { getBetaPopupClosed, setBetaPopupClosed } from 'src/adapters/store';
+import { parseQueryString } from '../libs/url_utils';
+
+const initialQueryParams = parseQueryString(window.location.search);
+
+const BetaInfoBox = () => {
+  const [closed, setClosed] = useState(getBetaPopupClosed());
+  const [client] = useState(initialQueryParams['client']);
+
+  const closeBetaPopup = () => {
+    setBetaPopupClosed();
+    setClosed(true);
+  };
+
+  const footer = <div className="alert-link">
+    <Button variant="tertiary" onClick={closeBetaPopup}>
+      {_('Do not display this message again')}
+    </Button>
+  </div>;
+
+  return (
+    // Show beta popup if browser language is not french, if popup hasn't already been closed, and if the user comes from an IA
+    !navigator.language.includes('fr')
+    && !closed
+    && client === 'search-ia-local'
+    &&
+    <Alert
+      className=""
+      title="Qwant Maps is in Beta!"
+      /* eslint-disable max-len */
+      description={_('This means that this version may have some bugs. We work very hard to improve Qwant Maps every day, while keeping your travels private.')}
+      /* eslint-enable max-len */
+      type="info"
+      onClose={closeBetaPopup}
+      footer={footer}
```

Comment:
No big deal, but you can also render the footer directly here, this is a common practice to pass JSX as prop.